airflow/providers/docker/operators/docker.py

```diff
@@ -291,6 +296,9 @@ def __init__(
             if skip_on_exit_code
             else []
         )
+        self.port_bindings = port_bindings or {}
```

Comment:
Is it really useful to coerce this to dict here? From what I can tell down below, we’re actually converting the value back to None anyway. So it may actually be easier to do `self.port_bindings = port_bindings or None` instead?

Reply:
yes sending an empty list/dict will work. @uranusjr